composer.json

```
@@ -32,6 +32,7 @@
         "drupal/auto_entitylabel": "3.0-beta4",
         "drupal/background_image_formatter": "1.7",
         "drupal/bootstrap_barrio": "4.30",
+        "drupal/bootstrap_utilities": "^1.0",
```

We'll want to pin to a specific version of the module.
```suggestion
        "drupal/bootstrap_utilities": "1.0",
```